doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -15761,6 +15761,8 @@ In EnergyPlus the relief and outdoor air dampers, the economizer, and any outdoo
 
 Of course any type of setpoint manager can be used to establish a temperature setpoint at the mixed air node. But the Mixed Air Setpoint Manager is likely to be the most useful.
 
+When optional inputs of Cooling Coil Inlet Node Name, Cooling coil Outlet Node Name, and Minimum Temperature at Cooling Coil Outlet Node are provided, the setpoint temperature at the mixed air node is calculated based on the maximum of both the reference setpoint temperature and minimum cooling coil outlet temperature, minus the cooling coil air temperature reduction and the supply fan air temperature rise if the supply fan placement is blow through. The setpoin temperature at the mixed air node is used to claculate the maximum outdoor air flow fraction in the corresponding Controller:OutdoorAir object. The outdoor air flow fraction is cappedd at the maximum outdoor air flow fraction to prevent cooling coil from freezing, even though the fraction is below the minimum outdoor air flow fraction.  
```

Comment:
Typos in 2nd sentence "The setpoin temperature" and "claculate" and "cappedd" and "prevent cooling" --> "prevent the cooling"